pkg/rpcclient/nep11/base.go

```diff
@@ -246,3 +247,71 @@ func UnwrapKnownProperties(m *stackitem.Map, err error) (map[string]string, erro
 	}
 	return res, nil
 }
+
+// TransferEventsFromApplicationLog retrieves all emitted TransferEvents from the
+// provided ApplicationLog.
```

Comment:
`[result.ApplicationLog]` to have a doc link.

Reply:
Fixed.